### PR TITLE
Various phpdoc fixes

### PR DESCRIPTION
**Pull request by braders**
Overview
----------------------------------------
Lots more phpdoc fixes.

Largely these handle the case where previously the type definition was incorrectly placed at the end of the `@param` line, rather than before the variable name.

Before
----------------------------------------
PHPdoc inaccuracies.

After
----------------------------------------
PHPdoc (more) accurare.

Comments
----------------------------------------
This is far from comprehensive - there are still lots of PHPdoc issues left after this set of changes.

### CODE REVIEW

File: CRM/Contact/BAO/ProximityQuery.php [hunk 1]
```diff
@@ -378,7 +389,6 @@ public static function fixInputParams(&$input) {
             }
           }
         }
-        return NULL;
```
Comment (braders) [inline CRM/Contact/BAO/ProximityQuery.php:381]: The return type was not being used anywhere, and the docblock was previously being contradictory. void ultimately felt more appropriate.

**Comment by colemanw** [comment]
Great cleanup, excellent work.
Jenkins is a bit grumpy about some of the changes though. If you click the "Details" link ^ and then click a few more times to drill down you get to this:
image

File: CRM/Core/BAO/EntityTag.php [hunk 2]
```diff
@@ -319,7 +320,7 @@ public static function getContactTags($contactID, $count = FALSE) {
 
     if ($count) {
       $dao->fetch();
-      return $dao->cnt;
+      return (int) $dao->cnt;
```
Comment (eileenmcnaughton) [inline CRM/Core/BAO/EntityTag.php:323]: Because cnt is sooo much shorter than count .....

Comment (colemanw) [reply CRM/Core/BAO/EntityTag.php:323]: Cuter too

Comment (totten) [reply CRM/Core/BAO/EntityTag.php:323]: I'm fairly certain that I've seen errors in the past when trying to use `count` as a column-alias in DAO queries. Never traced the cause - just made a mental note to go along with `cnt` as the typical work-around name.

But... in the current stack, it looks like `SELECT ... AS count` works fine. So I'm +1 for the `count`...

Comment (eileenmcnaughton) [reply CRM/Core/BAO/EntityTag.php:323]: @totten you do know my comments about always 'reading' variables applies - there is only one way to read cnt outloud

**Pull request merged**